hadoop-hdds/framework/src/main/java/org/apache/hadoop/ozone/audit/AuditLogger.java

```diff
@@ -155,6 +160,22 @@ public void appendOpLatencyNanos(long nanos) {
       append("opLatencyMs", TimeUnit.NANOSECONDS.toMillis(nanos));
     }
 
+    /**
+     * Appends pre-operation operation latency in milliseconds.
+     * @param millis Latency in nanoseconds.
+     */
+    public void appendPreOpLatencyMs(long millis) {
+      append("preOpLatencyMs", TimeUnit.NANOSECONDS.toMillis(millis));
```

Comment:
Name and param say "millis", but conversion is from nanos.

Reply:
Fixed.